mw-webapp/src/model/businessModel/DayReport.ts

```diff
@@ -33,9 +34,9 @@ export class DayReport {
   public problemsForCurrentPeriod: CurrentProblem[];
 
   /**
-   * Anything that student wants to say about work
+   * Array of student comments and their status
```

Comment:
Dont duplicate information it will be hard to maintain it. What if the `StudentComment` model changed or the status field changed etc.. Document only useful information that couldn't be placed in code (or if it is hard etc.).
in current case `Student comment` will be enough (yeah code is self-documented in this case)

Reply:
Done